corehq/apps/hqadmin/reports.py

```diff
@@ -317,3 +319,8 @@ def _format_date(self, date):
 
     def _hyperlink_diff_url(self, diff_url):
         return f'<a href="{diff_url}">Diff with previous</a>'
+
+    def _shorten_and_hyperlink_commit(self, commit_sha):
+        if commit_sha:
+            return f'<a href="https://github.com/dimagi/commcare-hq/commit/{commit_sha}">{commit_sha[:7]}</a>'
+        return None
```

Comment:
I don't think it's necessary to shorten the commit here - you can still link to the full thing.

Reply:
![image](https://user-images.githubusercontent.com/41345510/89204293-b8afcc80-d56a-11ea-89dd-b0e946b8229a.png)

This is what the shortened commit sha looks like. The hyperlink still takes you to the address with the full commit sha, but the shortening just looks cleaner in the table.

Reply:
Oh you're right, I misread that - your way is better.